parmed/structure.py

```diff
@@ -1293,6 +1292,104 @@ def split(self):
 
     #===================================================
 
+    def save(self, fname, kind=None, **kwargs):
```

Comment:
Format is a better choice. Kind was just the first word that came to mind when I was coding it.